share/eds/byzantine/pb/share.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package share.eds.byzantine.pb;
+
+option go_package="github.com/celestiaorg/celestia-node/share/eds/byzantine/pb";
```

Comment:
Do we need it?

Reply:
No really, since no more imports if this proto. Removed it